src/core_plugins/kibana/public/dashboard/styles/index.less

```diff
@@ -13,13 +58,12 @@
 dashboard-grid {
   display: block;
   margin: 0;
-  padding: 5px;
+  padding: 20px;
```

Comment:
Did we mean to change this here? It seems to add quite a bit of padding even when not in full-screen mode.